ToDoApp/ToDoApp/ToDoListViewModel.swift

```diff
@@ -81,4 +81,11 @@ final class ToDoListViewModel: ObservableObject {
             }
         }
     }
+    // Update the status of the task
+    func updateStatus(id: UUID) {
+        Task {
+            _ = await taskService.updateTaskStatus(id: id)
+//            self.fetchToDoTasks()
```

Comment:
It seems that this shouldn't be commented

Reply:
oh my bad, I commented it to debug the code but forgot to uncomment, and it's fixed now.